contributing/code/bc.rst

```diff
@@ -253,6 +257,10 @@ Change return type                                  Yes
 **Static Methods**
 Turn non static into static                         No
 Turn static into non static                         No
+**Constants**
+Add constant                                        Yes
+Remove constant                                     No
+Change value of a constant                          Yes [5]_
```

Comment:
I don't think it'll be usefull to document value change of a constant. It'll be more important to document expected value change for the argument (e.g. what has been done for the UrlGenerator in 2.8).